addons/l10n_in_stock/models/stock_picking.py

```diff
@@ -10,3 +10,12 @@ class StockPicking(models.Model):
     def _should_generate_commercial_invoice(self):
         super(StockPicking, self)._should_generate_commercial_invoice()
         return True
+
+    def _get_l10n_in_dropship_dest_partner(self):
+        pass
```

Comment:
If a method needs to be overridden we usually put a comment (docstring) that the method is to override and we return False.  (but it does not complain in the ci/style)  Actually pass has the same effect as return.